.go-version

```diff
@@ -1 +1 @@
-1.23.1
+1.23.2
```

Comment:
Would it be better to set here the same as in the go.mod (go directive) 1.22.8?

Reply:
This is the version used for building, right? I would use the latest one here.

Reply:
Exactly, this is going to be used in CI and also following the indications of the README to build the package-registry binary too: https://github.com/elastic/package-registry?tab=readme-ov-file#go-command